api_reference.md

```diff
@@ -35,9 +45,7 @@ The maximum text size per request is 3000 bytes.
     "toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes referred to
     as an **attribute** or **model attribute**.
 
-*   The API can return **model attribute scores** in different formats, known as
-    **score types**. The default score type is a probability score between 0 to
-    1, with higher values indicating greater likelihood of the attribute label.
+*   The API can return **model attribute scores** in different formats, known as **score types**. Currently, the only score type supported is a probability score between 0 and 1, with higher values indicating greater likelihood of the attribute    label.
```

Comment:
NIT: it looks like there are extra spaces between "attribute" and "label", although this doesn't seem to affect the formatted documentation

Reply:
I committed the fix. Merging now